src/components/animation/ParallelAnimationFactory.brs

```diff
@@ -0,0 +1,34 @@
+' @import /components/animator/AnimatorFactory.brs
+' @import /components/getProperty.brs
+' @import /components/rokuComponents/ParallelAnimation.brs
+
+' @class
+function ParallelAnimationFactory() as Object
+  prototype = {}
+
+  ' Creates ParallelAnimation component with one or more Animations.
+  ' @param {String} name
+  ' @param {Object} [options={}]
+  ' @param {Float} options.delay
+  ' @param {Boolean} options.repeat
+  ' @param {AnimatorFactory~Options} options.animations
+  ' @param {Node} element
+  ' @returns {Object} - ParallelAnimation component
+  prototype.createAnimation = function(name as String, options = {} as Object, element = invalid as Object) as Object
+    parallelAnimationNode = ParallelAnimation()
+    parallelAnimationNode.setFields({
+      id: name,
+      delay: getProperty(options, "delay", 0.001),
+      repeat: getProperty(options, "repeat", false),
+    })
+
+    factory = AnimatorFactory()
+    for each animationOption in options.animations.items()
+      parallelAnimationNode.appendChild(factory.createAnimation(animationOption.key, animationOption.value))
+    end for
```

Comment:
`AnimatorFactory.createAnimation` has such an interface: `function (name as String, options = {} as Object, element = Invalid as Object)`
So if you implement my suggestion (https://github.com/getndazn/kopytko-utils/pull/25/files#r1378563587) you could iterate through options.animations objects and code would look clearer:

```suggestion
    for each animation in options.animations
      parallelAnimationNode.appendChild(factory.createAnimation(animation.elementId, animation.animatorOptions))
    end for
```

Reply:
From the definition point of view clearer is:

```
{
  someAnimationConfig: {
     animations: {
       gradient: {
          duration:0.2
          easeFunction: "linear"
          fields: [...]
       }
     }
  }
}
```
vs
```
{
   someAnimationConfig: {
     animations: [
      {
        name: "gradient",
        options: {
          duration: 0.2,
          easeFunction: "linear",
          fields: [],
        }
      }
    ]
  }
}
```
Less code and more compact imho.